build/azure-pipelines.yml

```diff
@@ -0,0 +1,90 @@
+trigger:
+  branches:
+    include:
+    - master
+    - feature/*
+
+resources:
+   containers:
+     - container: windows
+       image: nventive/vs_build-tools:17.1.1
+
+variables:
+- name: NUGET_VERSION
+  value: 6.1.0

```

Comment:
Why already at 6.1 ?

Reply:
Based everything off an existing pipeline we're using for another project. Do you think we should be referencing a different version than this?

Reply:
With vs_build-tools:17.1.1, I think we need 6.1.0